docs/README.md

```diff
@@ -49,15 +53,37 @@ All you need to do to run Bellhop is deploy the solution and ensure you have the
 ## Required Tags for All Services
 Bellhop operates based on resource tags. Some of the required tags will be common between Azure services, and some tags will be specific to the resource you would like Bellhop to scale. Resource specific tags will be discussed in detail in the [Scaler Modules](/scaler/modules/README.md) section.
 
-Bellhop Common Tags:
+**Bellhop Common Tags:**
 ```
 resize-Enable = <bool> (True/False)
 resize-StartTime = <DateTime> (Friday 7PM)
 resize-EndTime = <DateTime> (Monday 7:30AM)
 ```
 
-_**NOTE: StartTime and EndTime are currently in UTC**_
-
+### StartTime & EndTime Tags
+What does 'StartTime' & 'EndTime' represent?
+- The `StartTime` tag represents when Bellhop will scale _**DOWN**_ the target resource (saves the current state)
+- The `EndTime` tag represents when Bellhop will scale _**UP**_ the target resource (reverts to saved state)
+
+Can you provide some examples of how to format the expected timestamp?
+- The expected format is `<DayOfWeek> <TimeOfDay>`
+    - `<TimeOfDay>` is the time, based on the 12HR clock, which can be simply the hour, or hour + minutes
+    - Hour Only: `resize-StartTime = Monday 7PM`
+    - Hour + Minutes: `resize-StartTime = Friday 9:30AM`
+
+Are the timestamps in a specific timezone, or can I select a timezone?
+- All timestamps are in UTC format
+- There are many complexities involved in supporting individual timezones, however this functionailty may come in the future
+    - Please feel free to discuss your specific requirements around this on our [GitHub Discussions](https://github.com/Azure/bellhop/discussions) page
+
+Can I have Bellhop resize Daily/Weekly/Monthly?
+- Currently, we do support using the word `Daily` in place of the `<DayOfWeek>` which will trigger the resize action on a daily basis
+- When using the `Daily` value, you must use `Daily` in both tags ( you cannot mix `Daily` and `<DayOfWeek>`)
+- Example usage:
+    - `resize-StartTime = Daily 8PM`
+    - `resize-EndTime = Daily 6AM`
```

Comment:
I would add some explanation below:
The effect of these two tags would be that the resource is scaled down between 8PM and 6AM, and scaled up again at 6AM.

Reply:
I added the sentence "The net effect of example tag combination above, is that the target resource would be scaled down every evening at 8PM, and then scaled back up every morning at 6AM (UTC time)" below the two tags.